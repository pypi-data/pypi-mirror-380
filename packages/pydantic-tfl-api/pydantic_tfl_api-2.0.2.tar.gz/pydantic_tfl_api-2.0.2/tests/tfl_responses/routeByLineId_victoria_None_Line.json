{
  "status_code": 200,
  "headers": {
    "Date": "Mon, 15 Jul 2024 15:40:58 GMT",
    "Content-Type": "application/json; charset=utf-8",
    "Content-Length": "442",
    "Connection": "keep-alive",
    "Cache-Control": "public, must-revalidate, max-age=43200, s-maxage=86400",
    "Via": "1.1 varnish",
    "Content-Encoding": "gzip",
    "Accept-Ranges": "bytes",
    "Age": "14662",
    "Access-Control-Allow-Headers": "Content-Type",
    "Access-Control-Allow-Methods": "GET,POST,PUT,DELETE,OPTIONS",
    "Access-Control-Allow-Origin": "*",
    "Api-Entity-Payload": "Line,RouteSection",
    "X-Backend": "api",
    "X-Cache": "HIT",
    "X-Cache-Hits": "5",
    "X-Cacheable": "Yes. Cacheable",
    "X-Frame-Options": "deny",
    "X-Proxy-Connection": "unset",
    "X-TTL": "86400.000",
    "X-TTL-RULE": "0",
    "X-Varnish": "2107610357 2104798358",
    "X-AspNet-Version": "4.0.30319",
    "X-Operation": "Line_LineRoutesByIdsByPathIdsQueryServiceTypes",
    "X-API": "Line",
    "CF-Cache-Status": "DYNAMIC",
    "Strict-Transport-Security": "max-age=31536000",
    "Set-Cookie": "_cfuvid=s1hEyjLFkCKxenTdZs9J7mjoO9eEHEtyqsMhqmv83MY-1721058058892-0.0.1.1-604800000; path=/; domain=.tfl.gov.uk; HttpOnly; Secure; SameSite=None",
    "Server": "cloudflare",
    "CF-RAY": "8a3adaa3af4b6418-LHR"
  },
  "url": "https://api.tfl.gov.uk/Line/victoria/Route",
  "content": "{\"$type\":\"Tfl.Api.Presentation.Entities.Line, Tfl.Api.Presentation.Entities\",\"id\":\"victoria\",\"name\":\"Victoria\",\"modeName\":\"tube\",\"disruptions\":[],\"created\":\"2024-07-09T13:08:01.79Z\",\"modified\":\"2024-07-09T13:08:01.79Z\",\"lineStatuses\":[],\"routeSections\":[{\"$type\":\"Tfl.Api.Presentation.Entities.MatchedRoute, Tfl.Api.Presentation.Entities\",\"name\":\"Walthamstow Central Underground Station - Brixton Underground Station\",\"direction\":\"inbound\",\"originationName\":\"Walthamstow Central Underground Station\",\"destinationName\":\"Brixton Underground Station\",\"originator\":\"940GZZLUWWL\",\"destination\":\"940GZZLUBXN\",\"serviceType\":\"Regular\",\"validTo\":\"2024-12-23T00:00:00Z\",\"validFrom\":\"2024-07-06T00:00:00Z\"},{\"$type\":\"Tfl.Api.Presentation.Entities.MatchedRoute, Tfl.Api.Presentation.Entities\",\"name\":\"Brixton Underground Station - Walthamstow Central Underground Station\",\"direction\":\"outbound\",\"originationName\":\"Brixton Underground Station\",\"destinationName\":\"Walthamstow Central Underground Station\",\"originator\":\"940GZZLUBXN\",\"destination\":\"940GZZLUWWL\",\"serviceType\":\"Regular\",\"validTo\":\"2024-12-23T00:00:00Z\",\"validFrom\":\"2024-07-06T00:00:00Z\"}],\"serviceTypes\":[{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Regular\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Regular\"},{\"$type\":\"Tfl.Api.Presentation.Entities.LineServiceTypeInfo, Tfl.Api.Presentation.Entities\",\"name\":\"Night\",\"uri\":\"/Line/Route?ids=Victoria&serviceTypes=Night\"}],\"crowding\":{\"$type\":\"Tfl.Api.Presentation.Entities.Crowding, Tfl.Api.Presentation.Entities\"}}"
}