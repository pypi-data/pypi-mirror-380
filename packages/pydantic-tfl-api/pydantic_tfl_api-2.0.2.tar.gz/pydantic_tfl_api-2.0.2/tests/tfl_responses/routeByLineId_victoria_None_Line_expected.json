[
  {
    "id": "victoria",
    "name": "Victoria",
    "modeName": "tube",
    "disruptions": [],
    "created": "2024-07-09T13:08:01.79Z",
    "modified": "2024-07-09T13:08:01.79Z",
    "lineStatuses": [],
    "routeSections": [
      {
        "routeCode": null,
        "name": "Walthamstow Central Underground Station - Brixton Underground Station",
        "direction": "inbound",
        "originationName": "Walthamstow Central Underground Station",
        "destinationName": "Brixton Underground Station",
        "originator": "940GZZLUWWL",
        "destination": "940GZZLUBXN",
        "serviceType": "Regular",
        "validTo": "2024-12-23T00:00:00Z",
        "validFrom": "2024-07-06T00:00:00Z"
      },
      {
        "routeCode": null,
        "name": "Brixton Underground Station - Walthamstow Central Underground Station",
        "direction": "outbound",
        "originationName": "Brixton Underground Station",
        "destinationName": "Walthamstow Central Underground Station",
        "originator": "940GZZLUBXN",
        "destination": "940GZZLUWWL",
        "serviceType": "Regular",
        "validTo": "2024-12-23T00:00:00Z",
        "validFrom": "2024-07-06T00:00:00Z"
      }
    ],
    "serviceTypes": [
      {
        "name": "Regular",
        "uri": "/Line/Route?ids=Victoria&serviceTypes=Regular"
      },
      {
        "name": "Night",
        "uri": "/Line/Route?ids=Victoria&serviceTypes=Night"
      }
    ],
    "crowding": {
      "passengerFlows": null,
      "trainLoadings": null
    }
  }
]